tests/utilities.py

```diff
@@ -286,3 +309,50 @@ def get_role_members(role_id):
             .run(db_connection)
         )
     return role_members
+
+
+def log_in(session, credentials_payload):
+    """Log in as the user with the given credentials for the given session
+
+    Args:
+        session:
+            object: current session object
+
+        credentials_payload:
+            dictionary: in the format of
+                {
+                    "id": "USERNAME OF USER",
+                    "password": "PASSWORD OF ASSOCIATED USER"
+                }
+    """
+    response = session.post(
+        "http://rbac-server:8000/api/authorization/", json=credentials_payload
+    )
+    sleep(3)
+    return response
```

Comment:
If we don't need to return the response we don't have to return it. This way you won't have unused auth_response variables